pkg/controller/controller.go

```diff
@@ -119,10 +119,10 @@ func RegisterInformers(c *config.Config, notifiers []notify.Notifier) {
 	utils.DynamicKubeInformerFactory.Start(stopCh)
 
 	sigterm := make(chan os.Signal, 1)
-	signal.Notify(sigterm, syscall.SIGTERM, syscall.SIGINT, syscall.SIGKILL, syscall.SIGQUIT, syscall.SIGSTOP)
+	signal.Notify(sigterm, syscall.SIGTERM, syscall.SIGINT, syscall.SIGQUIT)
```

Comment:
Shouldn't we handle SIGKILL and SIGSTOP?

Reply:
Unfortunately we cannot handle them and this is what the linter reported:

```
pkg/controller/controller.go:122:58: SA1016: syscall.SIGKILL cannot be trapped (did you mean syscall.SIGTERM?) (staticcheck)
	signal.Notify(sigterm, syscall.SIGTERM, syscall.SIGINT, syscall.SIGKILL, syscall.SIGQUIT, syscall.SIGSTOP)
```

See: https://staticcheck.io/docs/checks#SA1016